### PR TITLE
Drop custom question listing

### LINKED ISSUE #6575: Drop custom template for question listings in favor of standard notes template
We have this custom template:

https://github.com/publiclab/plots2/blob/master/app/views/questions/_new_question.html.erb

For pages like this, displaying questions:

https://publiclab.org/tag/question:water-quality

However, we've really refined the regular listings here, visually:

https://publiclab.org/tag/water-quality

So, let's just use the same template, instead of switching. The template at top can be deleted. Then, we can just use this section:

https://github.com/publiclab/plots2/blob/bfe70900de941a78c4fab0c82a128ad46433795e/app/views/tag/show/_tab_content.html.erb#L4-L13

And remove the condition "if type == note"

And then, remove the section that's set up just for questions:

https://github.com/publiclab/plots2/blob/bfe70900de941a78c4fab0c82a128ad46433795e/app/views/tag/show/_tab_content.html.erb#L15-L36

That will simplify the code and make it more maintainable and consistent! We'd love help with this one!

### PR DESCRIPTION
**Pull request by sssash18**
Fixes #6575  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 
Screenshot 2020-01-18 at 4 43 15 PM
] 
Thanks!

### CODE REVIEW

**Comment by sssash18** [comment]
@Uzay-G After removing the tests suggested by you , then also the tests are failing.

File: test/functional/tag_controller_test.rb [hunk 1]
```diff
@@ -570,24 +562,6 @@ def setup
...
-    tag = tags(:question)
-
-    get :show_for_author, params: { id: tag.name, author: 'jeff' }
-
-    selector = css_select '#asked-tab.active'
-    assert_equal selector.size, 1
-    assert_select '#answered-tab', 1
-  end
-
-  test 'should list answered questions' do
```
Comment (Uzay-G) [inline test/functional/tag_controller_test.rb]: Sorry, you need to remove the test at line 564 of this file and not this one. Thanks

Comment (sssash18) [reply test/functional/tag_controller_test.rb]: No problem ,now it's working fine

**Comment by jywarren** [comment]
Hi! Can you highlight or screenshot where (on what pages, say, on the live site) there will no longer be answered/unanswered questions? Thank you!

**Comment by sssash18** [comment]
> Hi! Can you highlight or screenshot where (on what pages, say, on the live site) there will no longer be answered/unanswered questions? Thank you!

@jywarren according to me it will happen with the questions page as we only removed the template associated with it.Thanks

**Comment by sssash18** [comment]
@jywarren Any changes you would like to suggest?

**Comment by sssash18** [comment]
@jywarren @cesswairimu @SidharthBansal Anything here that needs to be changed or is it ready for merge?

**Comment by cesswairimu** [comment]
restarting travis

**Comment by jywarren** [comment]
Hi, unfortunately i think this is now out of date, as we've made so many changes to the tag pages! I'm so sorry for not responding sooner. If you'd like to refactor this and rebase this i'm happy to take another look, but i'm not sure it's needed any more? 

Please forgive me for being slow about this, we're only just now catching up with everything! Your help is deeply appreciated!!!

**Pull request closed without merging**